mesonbuild/scripts/clangtidy.py

```diff
@@ -26,5 +33,34 @@ def run(args: T.List[str]) -> int:
     srcdir = Path(options.sourcedir)
     builddir = Path(options.builddir)
 
-    run_func = run_clang_tidy_fix if options.fix else run_clang_tidy
-    return run_tool('clang-tidy', srcdir, builddir, run_func, builddir)
+    tidyexe = detect_clangtidy()
+    if not tidyexe:
+        print('Could not execute clang-tidy "%s"' % ' '.join(tidyexe))
```

Comment:
We don't use the `str % filler` format, either fstrings or str.format: `f'{Could not execute ... {" ".join(applyexe}'`, or `'Could not execute ...'.format(' '.join(applyexe))`